tests/swift-sdk-swift-tests/IterableUtilTests.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Created by Jay Kim on 12/1/20.
```

Comment:
From now on.... we should not add Author attribution anymore. Please let me know if I also put my name by mistake.